docs/security.md

```diff
@@ -31,7 +31,32 @@ class UserController extends Controller
      *
      * Creates new user or returns already existing user by email.
      */
-     #[OpenApi\Operation(security: 'BearerTokenSecurityScheme')]
+     #[OpenApi\Operation()]
+     #[OpenApi\SecurityRequirement('BearerTokenSecurityScheme')]
+    public function store(Request $request)
```

Comment:
I know I'm nitpicking, but I just noticed there's a missing indentation here. 😅 Could you fix it?